docs/en-US/component/collapse.md

```diff
@@ -62,15 +72,17 @@ collapse/customization
 
 ### Collapse Item Attributes
 
-| Name     | Description                        | Type                  | Default |
-| -------- | ---------------------------------- | --------------------- | ------- |
-| name     | unique identification of the panel | ^[string] / ^[number] | —       |
-| title    | title of the panel                 | ^[string]             | ''      |
-| disabled | disable the collapse item          | ^[boolean]            | false   |
+| Name     | Description                        | Type                     | Default |
+| -------- | ---------------------------------- | ------------------------ | ------- |
+| name     | unique identification of the panel | ^[string] / ^[number]    | —       |
+| title    | title of the panel                 | ^[string]                | ''      |
+| icon     | icon of the collapse item          | ^[string] / ^[Component] | -       |
```

Comment:
Maybe only one `icon` slot is enough, which can meet the scenarios corresponding to the attribute `icon`, what do you think?

Reply:
@btea Unlike the icon slot, the `icon` attributes can apply the style of the default icon, so that the icon can be set quickly without having to redefine the style.

Reply:
Make sense.